src/js/constants/ContainerConstants.js

```diff
@@ -1,22 +1,22 @@
 const ContainerConstants = {
-  NETWORK: {
+  network: Object.freeze({
```

Comment:
But @aldipower – why not at least do 

```
const ContainerConstants = Object.freeze({
// ....
```
?

I see why you did this: `const` would effectively only prevent us from re-assigning it (ie stuff like `ContainerConstants.newProperty = true;` won't be prevented at all) but I really don't love all these `Object.freeze` calls littering our object definitions...